hayabusa/hayabusa.go

```diff
@@ -38,18 +44,31 @@ func Genesis(config *Config) *genesis.CustomGenesis {
 		Build()
 }
 
-func StartNetwork(config *Config) (*httpclient.Client, *network.CustomNetwork, func(), error) {
+func StartNetwork(config *Config) (*httpclient.Client, environments.Actions, func(), error) {
 	if config.Nodes < 2 {
 		return nil, nil, nil, fmt.Errorf("at least 2 nodes are required")
 	}
 	repo := "git@github.com:vechain/thor.git"
-	var net *network.CustomNetwork
+
+	// reimplement this logic
 	workingDir, ok := os.LookupEnv("THOR_WORKING_DIR")
+	var thorBuilder *thorbuilder.Config
 	if ok {
-		net = network.NewCustomWithRepoAndDownloadPath(repo, workingDir, config.Debug)
+		thorBuilder = &thorbuilder.Config{
+			BuildConfig: &thorbuilder.BuildConfig{
+				ExistingPath: workingDir,
+				DebugBuild:   true,
```

Comment:
```suggestion
				DebugBuild:   config.Debug,
```

We default to false for GHA as the builds are less stable